src/utils/values.h

```diff
@@ -190,6 +190,28 @@ char *sr_val_to_str(const sr_val_t *value);
  */
 int sr_val_to_buff(const sr_val_t *value, char buffer[], size_t size);
 
+/**
+ * @brief Similar to ::sr_get_item but only gets configuration data from given data tree.
```

Comment:
Comment is wrong and insufficient. It should first say what it does - basically transforms `struct lyd_node` to `sr_val_t`. Then the intentional usage should be described, the one you mentioned, that if you want to work with the structures but do not want to call `sr_get_item(s)` many times because of performance, you can call `sr_get_data()` once and then use this function.